packages/animations/example/lib/shared_axis_transition.dart

```diff
@@ -14,11 +14,11 @@ class SharedAxisTransitionDemo extends StatefulWidget {
 }
 
 class _SharedAxisTransitionDemoState extends State<SharedAxisTransitionDemo> {
-  SharedAxisTransitionType _transitionType =
+  SharedAxisTransitionType? _transitionType =
```

Comment:
It is odd that this field is nullable.

Reply:
It has to be nullable since the radio button's `onChanged` callback is `final ValueChanged<T?>? onChanged`